stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -298,6 +322,11 @@ public void setSourceTypeData(Map<String, Object> sourceTypeData) {
         mSourceTypeData = sourceTypeData;
     }
 
+    public void setTypeRaw(@NonNull @Size(min = 1) String typeRaw) {
+        mTypeRaw = typeRaw;
+        setType(UNKNOWN);
+    }
+
     public void setType(@SourceType String type) {
         mType = type;
```

Comment:
As things are, it doesn't matter, since if `mType` is not unknown, then it is the source of truth (always) -- so setting the raw/custom type says "I am deciding what the type should be, and all other declarations are invalid."

If I move to using the rawType as what gets sent down, then it might make more sense to do that.